.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+npx lint-staged
```

Comment:
Why?

Reply:
I run the npx mrm@2 lint-staged command twice and I think that's why this problem occurred